synapseAnnotations/data/ngs.json

```diff
@@ -109,6 +109,130 @@
         "description": "",
         "source": ""
       }
+   ]
+  },
+  {
+    "name": "libraryPrepMethod",
+    "description": " method by which library was prepared",
+    "columnType":"STRING",
+    "maximumSize": 256,
+    "enumValues" : [
+      {
+        "value" : "10x",
```

Comment:
10x Genomics? isn't this from a specific company?

Reply:
It's what HCA has, so keeping it.